rclpy/src/rclpy/_rclpy.c

```diff
@@ -713,9 +713,19 @@ rclpy_send_request(PyObject * Py_UNUSED(self), PyObject * args)
   assert(convert_from_py != NULL &&
     "unable to retrieve convert_from_py function, type_support mustn't have been imported");
 
+  PyObject * pydestroy_ros_message = PyObject_GetAttrString(pymetaclass, "_DESTROY_ROS_MESSAGE");
+
+  typedef void * (* destroy_ros_message_signature)(void *);
+  destroy_ros_message_signature destroy_ros_message =
+    (destroy_ros_message_signature)PyCapsule_GetPointer(pydestroy_ros_message, NULL);
+
+  assert(destroy_ros_message != NULL &&
+    "unable to retrieve destroy_ros_message function, type_support mustn't have been imported");
```

Comment:
opened a separate issue for this because this is not tied to this PR (c.f. https://github.com/ros2/rclpy/issues/83)